fibonacci/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <iomanip>
+
+using namespace std;
+
+long double fib[100000]; int n;
```

Comment:
Check what 200L might have mean in declaration.

CLue: http://www.math.uni.opole.pl/~aczubak/c/Wyklad.04.pdf